src/main/java/org/ssor/boss/core/transfer/UpdateUserInput.java

```diff
@@ -12,32 +12,33 @@
 /**
  * Represents the data of a User that can be updated.
  * <p>
- *   User profiles contain their username, email, and password. User profiles don't typically contain their account
- *   holder information. This information should be updated or changed separately, if it is possible for a user to do so.
+ * User profiles contain their username, email, and password. User profiles
+ * don't typically contain their account holder information. This information
+ * should be updated or changed separately, if it is possible for a user to do
+ * so.
  * </p>
  */
 @Data
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class UpdateUserInput
-{
-  private int userId;
+public class UpdateUserInput {
```

Comment:
@christianangeles Here too 
```suggestion
public class UpdateUserInput 
{
```